astroquery/gaia/core.py

```diff
@@ -194,6 +196,11 @@ def load_data(self, ids, *, data_release=None, data_structure='INDIVIDUAL',
             For GAIA DR4, possible values will be ['EPOCH_PHOTOMETRY', 'RVS', 'XP_CONTINUOUS', 'XP_SAMPLED',
             'MCMC_GSPPHOT', 'MCMC_MSC', 'EPOCH_ASTROMETRY', 'RV_EPOCH_SINGLE', 'RV_EPOCH_DOUBLE', 'RVS_EPOCH' or
             'RVS_TRANSIT']
+        linking_parameter : str, optional, default None, valid values: SOURCE_ID, TRANSIT_ID, IMAGE_ID
+            By default, all the identifiers are considered as source_id
+            SOURCE_ID: the identifiers are considered as source_id
```

Comment:
Why to have `SOURCE_ID` as the default then, instead of None?

Reply:
Yes, you are right it is more convenient to set the default value to  SOURCE_ID